packages/core/src/contrib/Queries.ts

```diff
@@ -281,6 +282,145 @@ export const rectLineDist = (
+    const c4 = and(and(c1, c2), c3);
+    const c5 = not(c1);
+    const c6 = not(c2);
+    const c7 = not(c3);
+    const c8 = and(and(c5, c6), c7);
+    const negEss = mul(-1, ess);
+    ess = ifCond(or(c4, c8), negEss, ess);
+    // last line to match for loop in code we are borrowing from
+    j = i;
+  }
+  return mul(ess, sqrt(d));
+};
+
+// /**
+//  * Return the signed distance from the origin to the Minkowski sum of `rect` and
+//  * the negative of `line` (that is, `start` and `end` points both multiplied by
+//  * `-1`).
+//  */
+// export const rectLineDist = (
+//   rect: { bottomLeft: ad.Pt2; topRight: ad.Pt2 },
+//   line: { start: ad.Pt2; end: ad.Pt2 }
+// ): ad.Num => {
+//   const {
+//     bottomLeft: [rx0, ry0],
+//     topRight: [rx1, ry1],
+//   } = rect;
+//   const {
+//     start: [lxs, lys],
+//     end: [lxe, lye],
+//   } = line;
+//
+//   const px = gt(lxs, lxe);
+//   const py = gt(lys, lye);
+//
+//   // 0 means the negative is lesser, 1 means the negative is greater
+//   const lx0 = ifCond(px, lxs, lxe);
+//   const lx1 = ifCond(px, lxe, lxs);
+//   const ly0 = ifCond(py, lys, lye);
+//   const ly1 = ifCond(py, lye, lys);
+//
+//   const x00 = sub(rx0, lx0);
+//   const x01 = sub(rx0, lx1);
+//   const x10 = sub(rx1, lx0);
+//   const x11 = sub(rx1, lx1);
+//
+//   const y00 = sub(ry0, ly0);
+//   const y01 = sub(ry0, ly1);
+//   const y10 = sub(ry1, ly0);
+//   const y11 = sub(ry1, ly1);
+//
+//   const p = xor(px, py); // true iff negative slope
+//
+//   // if `p`:
+//   //
+//   // 4 - 3
+//   // |    \
+//   // 5     2
+//   //  \    |
+//   //   0 - 1
+//   //
+//   // point 0 is `ops.vsub([rx0, ry0], [lx1, ly0])`
+//   // point 1 is `ops.vsub([rx1, rx0], [lx1, ly0])`
+//   // point 2 is `ops.vsub([rx1, ry1], [lx1, ly0])`
+//   // point 3 is `ops.vsub([rx1, ry1], [lx0, ly1])`
+//   // point 4 is `ops.vsub([rx0, ry1], [lx0, ly1])`
+//   // point 5 is `ops.vsub([rx0, ry0], [lx0, ly1])`
+//
+//   // if not `p`:
+//   //
+//   //   4 - 3
+//   //  /    |
+//   // 5     2
+//   // |    /
+//   // 0 - 1
+//   //
+//   // point 0 is `ops.vsub([rx0, ry0], [lx0, ly0])`
+//   // point 1 is `ops.vsub([rx1, ry0], [lx0, ly0])`
+//   // point 2 is `ops.vsub([rx1, ry0], [lx1, ly1])`
+//   // point 3 is `ops.vsub([rx1, ry1], [lx1, ly1])`
+//   // point 4 is `ops.vsub([rx0, ry1], [lx1, ly1])`
+//   // point 5 is `ops.vsub([rx0, ry1], [lx0, ly0])`
+//
+//   const x0 = ifCond(p, x01, x00);
+//   const x1 = ifCond(p, x11, x10);
+//   const x2 = x11;
+//   const x3 = ifCond(p, x10, x11);
+//   const x4 = ifCond(p, x00, x01);
+//   const x5 = x00;
+//
+//   const y0 = y00;
+//   const y1 = y00;
+//   const y2 = ifCond(p, y10, y01);
+//   const y3 = y11;
+//   const y4 = y11;
+//   const y5 = ifCond(p, y01, y10);
+//
+//   return convexPolygonOriginSignedDistance([
+//     [x0, y0],
+//     [x1, y1],
+//     [x2, y2],
+//     [x3, y3],
+//     [x4, y4],
+//     [x5, y5],
+//   ]);
+// };
```

Comment:
This commented-out code seems like just a duplication of the definition of `rectLineDist` above; could you delete it?

```suggestion
```

Reply:
Sure thing—not sure what was going on there…